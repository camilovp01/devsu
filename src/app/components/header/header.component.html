<header>
  <img src="images/devsu_logo.jpg" alt="logo"  />
</header>
