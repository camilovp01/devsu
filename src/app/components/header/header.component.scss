.header {
  text-align: center;
  height: 80px;
  &__logo {
    width: 100px;
    object-fit: contain;
    cursor: pointer;
  }
}
