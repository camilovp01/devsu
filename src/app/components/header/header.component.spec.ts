import { ComponentFixture, TestBed } from '@angular/core/testing';

import { RouterModule } from '@angular/router';
import { HeaderComponent } from './header.component';

describe('HeaderComponent', () => {
  let component: HeaderComponent;
  let fixture: ComponentFixture<HeaderComponent>;
  let compiled: HTMLElement;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [HeaderComponent, RouterModule.forRoot([])],
    }).compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(HeaderComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should match the snapshot', () => {
    expect(compiled).toMatchSnapshot();
  });
});
