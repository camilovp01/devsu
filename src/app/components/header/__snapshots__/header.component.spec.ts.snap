// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent should match the snapshot 1`] = `
<div
  id="root1"
>
  <header>
    <img
      alt="logo"
      src="images/devsu_logo.jpg"
    />
  </header>
</div>
`;
